modules/ols-attaching-a-resource-object-to-your-query.adoc

```diff
@@ -0,0 +1,29 @@
+// This module is used in the following assemblies:
+// ols-using-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ols-attaching-a-resource-object-to-your-query_{context}"]
+= Attaching a resource object to your query
+
+This procedure explains how to attach a resource object to provided additional context for your chat query. 
+
+.Procedure
+
+. Navigate to a supported resource in the {ocp-product-title} web console. For example, click *Workloads* -> *Pods* and then click the name of a pod.
++
+[NOTE]
+====
+If you bring up the user interface using any of the *Ask OpenShift Lightspeed* menu options, the attachment option is present.
```

Comment:
I think we can remove this -- the `+` button I think is _always_ there, even for unsupported pages. Would need to double check with Andy.

Reply:
done